spec.bs

```diff
@@ -1104,10 +1099,11 @@ To <dfn>build bid generators map</dfn> given an [=auction config=] |auctionConfi
 
 <div algorithm="generate a bid">
 
-To <dfn>generate a bid</dfn> given an [=ordered map=] |allTrustedBiddingSignals|, a
-[=string=] |auctionSignals|, a {{BiddingBrowserSignals}} |browserSignals|, a [=string=]
-|perBuyerSignals|, a [=duration=] |perBuyerTimeout| in milliseconds, a [=currency tag=] |expectedCurrency|,
-an [=interest group=] |ig|, and a [=moment=] |auctionStartTime|:
+To <dfn>generate a bid</dfn> given an [=ordered map=] |allTrustedBiddingSignals|, a [=string=]
+|auctionSignals|, a {{BiddingBrowserSignals}} |browserSignals|, a [=string=]-or-null |perBuyerSignals|,

```

Comment:
Per my understanding, |buyerDirectFromSellerSignals| is a webIDL type, so it can use {{undefined}}.
|perBuyerSignals| is a [=string=], which is an infra type, so paired it with null, instead of webIDL {{undefined}}. Because [=string=] can be directly converted to a JS value before JS evaluation, so used it instead of webIDL {{USVString}}, though they are quite interchangable IIRC.